config/backup/config.go

```diff
@@ -27,6 +30,19 @@ func Configure(p *config.Provider) { //nolint:gocyclo
 		r.References["plan_id"] = config.Reference{
 			TerraformName: "aws_backup_plan",
 		}
+		r.TerraformCustomDiff = func(diff *terraform.InstanceDiff, state *terraform.InstanceState, config *terraform.ResourceConfig) (*terraform.InstanceDiff, error) {
+			if diff == nil || diff.Empty() || diff.Destroy || diff.Attributes == nil {
+				return diff, nil
+			}
+			for key, attrDiff := range diff.Attributes {
+				if strings.HasPrefix(key, "condition.") && strings.HasSuffix(key, ".#") {
```

Comment:
nit: We may consider using a regex for a comprehensive check, but this is not necessary for now. I just wanted to note this. This state looks good to me.